src/pretix/base/models/event.py

```diff
@@ -330,6 +330,21 @@ def get_payment_providers(self) -> dict:
                 providers[pp.identifier] = pp
         return providers
 
+
+    @property
+    def get_event_microdata(self):
+        import json
+
+        eventdict = {"@context": "http://schema.org", "@type": "Event"}
+        eventdict["location"] = {"@type": "Place",
+                                 "address": str(self.location)}
+        eventdict["startDate"] = str(self.date_from)
+        eventdict["name"] = str(self.name)
+        eventdict["endDate"] = str(self.date_to)
```

Comment:
We should only set this property if ``self.settings.show_date_to`` is True and ``self.date_to`` is not ``None``, otherwise we do not know the end date, i.e.
````
if self.date_to and self.settings.show_date_to:
    if self.settings.show_times:
        eventdict["endDate"] = self.date_to.isoformat()
    else:
        eventdict["endDate"] = self.date_to.date().isoformat()
```